src/main/protobuf/extvm/msg.proto

```diff
@@ -1,6 +1,17 @@
 syntax = "proto3";
 package io.iohk.ethereum.extvm;
 
+message Hello {
+  string version = 1;
+
+  // A blockchain related configuration for the VM for the entire session.
+  // Currently only Mantis-EVM is supported. Other VMs may ingore this
+  // or may required different configurations.
+  oneof config {
```

Comment:
can we have a separate message containing the oneof that is shared in common between Hello and CallContext? something like:
```
message Hello {
  string version = 1;
  BlockchainConfig config = 2;
}

message CallContext {
  ...
  BlockchainConfig config = 11;
}

message BlockchainConfig {
  oneof config {
    EthereumConfig ethereumConfig = 1;
  }
}
```

There would seem to be a bug in the code that generates ocaml code from a .proto file, and it incorrectly generates the fields of oneof fields as required instead of optional, which means that we cannot support creating an instance of CallContext in OCAML without first creating an EthereumConfig, which is somewhat undesirable for testing purposes.

Another alternative would be to add a second case to both oneofs with a IeleConfig that is just an empty message; this could also work. I'm comfortable with whichever you'd prefer, I just don't want us to have to create an entire EthereumConfig in order to test the IELE side of the connection because of this bug, and trying to get the bug fixed upstream will probably take quite some time.

Reply:
I would go with the empty `IeleConfig` - it fits what I had in mind here, and we may still end up needing it non-empty.

Reply:
@dwightguth The way Scala code generator works for `oneof` messages is that it always creates a class `Empty`. So for this field we have 2 possibilities: it will either be a `Hello.Config.EthereumConfig` or a `Hello.Config.Empty` - which is how we achieve optionality. Are you sure there isn't something like that in OCaml?

Reply:
I'm sure, yes. The protobuf docs say that oneof is an optional field, so the ocaml behavior is definitely a bug, but unless you want me to work on getting it fixed upstream, we need to work around it.

Reply:
I've added empty `IeleConfig` as another option. 
You said you cannot create a `CallContext`, but does the bug also mean you decode a `CallContext` if sent to you with empty `config` field?

Reply:
I'm not sure what would happen if you tried to decode it, but my guess is it would either decode incorrectly or throw some kind of exception.